x/posts/internal/types/poll_data.go

```diff
@@ -0,0 +1,269 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// ---------------
+// --- PollData
+// ---------------
+
+// PollData contains the information of a poll that is associated to a post
+type PollData struct {
+	Title                 string      `json:"title"`                   // Describes what poll is about
+	ProvidedAnswers       PollAnswers `json:"provided_answers"`        // Lists of answers provided by the creator
+	EndDate               time.Time   `json:"end_date"`                // RFC3339 date at which the poll will no longer accept new answers
+	Open                  bool        `json:"open"`                    // Tells if the poll is still accepting answers
```

Comment:
I think a better json key would be `is_open` as it's a boolean value

```suggestion
	Open                  bool        `json:"is_open"`                    // Tells if the poll is still accepting answers
```